### DISCUSSION TITLE
"Use region at own risk"

### DISCUSSION THREAD

**Discussion by MikeLanglois** [redphx/better-xcloud#516] [General]
Hey all,

What exactly are the risks to using the region selector in app? I am in a country that doesnt support xcloud at the moment and want to give it a go, but unsure of what the risks are?

**Comment by isaidj** [comment]
The risks are the same as using a VPN, Microsoft has not said anything about bans for using a VPN.

**Reply by MikeLanglois** [reply]
Ok cool I thought that might be it, but wanted to clarify thanks
